addons/mail/static/src/discuss/call/common/@types/models.d.ts

```diff
@@ -2,24 +2,37 @@ declare module "models" {
     import { Rtc as RtcClass } from "@mail/discuss/call/common/rtc_service";
     import { RtcSession as RtcSessionClass } from "@mail/discuss/call/common/rtc_session_model";
 
+    export interface Rtc extends RtcClass {}
+    export interface RtcSession extends RtcSessionClass {}
+
     export interface ChannelMember {
-        rtcSession: RtcSession,
+        rtcSession: RtcSession;
+    }
+    export interface Persona {
+        currentRtcSession: RtcSession;
+    }
+    export interface Settings {
+        getVolume: (rtcSession: unknown) => boolean;
     }
     export interface Store {
-        allActiveRtcSessions: RtcSession[],
-        ["discuss.channel.rtc.session"]: typeof RtcSessionClass,
-        nextTalkingTime: number,
-        ringingThreads: Thread[],
+        allActiveRtcSessions: RtcSession[];
+        "discuss.channel.rtc.session": RtcSession;
+        nextTalkingTime: number;
+        ringingThreads: Thread[];
+        rtc: Rtc;
+        Rtc: Rtc;
```

Comment:
I don't think the instance and the class are supposed to have the same type.
I don't see where `Rtc` is defined actually.